tests/test_obo_reading.py

```diff
@@ -17,8 +17,9 @@ def test_read_taxrank_file():
     with open(path, 'rt') as read_file:
         taxrank = obonet.read_obo(read_file)
     assert len(taxrank) == 61
-    assert taxrank.node['TAXRANK:0000001']['name'] == 'phylum'
-    assert 'NCBITaxon:kingdom' in taxrank.node['TAXRANK:0000017']['xref']
+    # It looks like networkx has changed the name of the node variable to nodes -- EST 2020-09-25
+    assert taxrank.nodes['TAXRANK:0000001']['name'] == 'phylum'
```

Comment:
looks like neworkx 2.4 [removed](https://networkx.github.io/documentation/stable/release/release_2.4.html) `G.nodes`, which had been marked as deprecated in 2.1. 

not sure why Travis CI builds are not showing up on GitHub for this pull request, but see the failures at https://travis-ci.org/github/dhimmel/obonet/builds/730420444. We support both networkx 1.x and 2.x.

You could add `pytest.importorskip("networkx", minversion="2.0")` to this test to skip it on 1.x builds. Removing 1.x support might be something we do soon anyways.

Reply:
The original (taxrank.node) fails with networkx 2.5:

Reply:
> The original (taxrank.node) fails with networkx 2.5:

yes. we should change to using `.node`, although this will cause failure for the networkx 1.x builds. Therefore, I was thinking we could just skip the taxrank tests if networkx < 2

Reply:
taxrank.node fails for me when using networkx 2.5, but I left the failing test as it was originally.